lib/puppet/context/trusted_information.rb

```diff
@@ -45,7 +45,7 @@ def self.remote(authenticated, node_name, certificate)
     if authenticated
       extensions = {}
       if certificate.nil?
-        Puppet.info('TrustedInformation expected a certificate, but none was given.')
+        Puppet.info(_('TrustedInformation expected a certificate, but none was given.'))
```

Comment:
TrustedInformation seems like a keyword, should we leave a comment to leave it out of translation?